app/views/home/index.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1>Home#index</h1>
-<p>Find me in app/views/home/index.html.erb</p>
+
+
+<div class="jumbotron">
+   <div class="container">
+     <h1>Welcome!</h1>
+     <p>Login to App.</p>
+     <p>
+     <%= link_to 'Login', new_user_session_path, class: 'btn btn-primary btn-lg' %>
```

Comment:
@saitheja12 - what if user is already logged in? We need to show `logout` button